kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -79,7 +106,7 @@ public void setProcurementCardGroupService(GroupService procurementCardGroupServ
     public boolean rerouteProcurementCardDocuments() {
         
         // check for reconciler group rerouting
-        boolean groupRerouting = parameterService.getParameterValueAsBoolean(ProcurementCardRerouteDocumentsStep.class, REROUTE_PCDO_DOCUMENTS_IND_PARM_NM);
+        boolean groupRerouting = parameterService.getParameterValueAsBoolean(ProcurementCardRerouteDocumentsStep.class, KFSParameterKeyConstants.ProcurementCardParameterConstants.REROUTE_PCDO_DOCUMENTS_IND_PARM_NM);
         
```

Comment:
Please add line here and refactor for loop accordingly to use this value:
String reconcilerGroups = KFSParameterKeyConstants.ProcurementCardParameterConstants.RECONCILER_GROUPS_TO_REROUTE_PARM_NM).getValue();
This will improve performance since the parameter is static and is accessed twice for every document in the for loop.